Lib/socketserver.py

```diff
@@ -531,6 +542,10 @@ def server_activate(self):
         # No need to call listen() for UDP.
         pass
 
+    def server_shutdown(self):
```

Comment:
In my opinion, it is worth leaving this method as `pass`. How this is done with the `server_activate` method since this is an unobvious behavior of the method.

Reply:
Fixed